common/changes/@microsoft/rush/credential-env-var_2021-05-04-10-33.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Change the environment variables used to customize cloud build cache settings",
```

Since this is a breaking change, I think the CHANGELOG.md notice should give a little more detail.
```suggestion
      "comment": "For the experimental build cache feature, eliminate the RUSH_BUILD_CACHE_WRITE_CREDENTIAL environment variable; it is replaced by several new variables RUSH_BUILD_CACHE_CREDENTIAL, RUSH_BUILD_CACHE_WRITE_ALLOWED, and RUSH_BUILD_CACHE_ENABLED",
```